crowbar_framework/app/assets/stylesheets/content/footer.scss

```diff
@@ -1,8 +1,10 @@
 footer {
-  color: #999;
+  position: absolute;
```

Comment:
Properties should be ordered bottom, color, font-size, position, text-align, width